components/executors/dind/patches/executor.ConfigMap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: executor-config
+# Refer to https://docs.sourcegraph.com/admin/executors/deploy_executors_binary#step-2-setup-environment-variables on how to populate these variables
+data:
+  EXECUTOR_FRONTEND_URL: "http://sourcegraph.example.com"
+  EXECUTOR_FRONTEND_PASSWORD: "our-shared-secret"
+  EXECUTOR_MAXIMUM_NUM_JOBS: "8"
+  # Used configure which queues Executors will process.
+  # Can be "batches" or "codeintel"
+  EXECUTOR_QUEUE_NAME: ""
```

Comment:
Do we want to add `EXECUTOR_QUEUE_NAMES` here?

Reply:
I'll add that as an option